### DISCUSSION TITLE
SSR not working on cloudflare pages

### DISCUSSION THREAD

**Discussion by myyrakle** [sveltejs/kit#9522] [Q&A] [answered]
From what I see in the docs, everything should work even if I use sveltekit in cloudflare Pages.

However, when deploying to cloudflare pages, only SSR does not work.
I think the data fetch executed by load doesn't work properly.

In the local environment, 'vite build & vite preview' works well.

The default settings followed the official documentation faithfully.

// svelte.config.js
```
import { vitePreprocess } from '@sveltejs/kit/vite';
import adapter from '@sveltejs/adapter-cloudflare';

/** @type {import('@sveltejs/kit').Config} */
const config = {
	// Consult https://kit.svelte.dev/docs/integrations#preprocessors
	// for more information about preprocessors
	preprocess: vitePreprocess(),

	kit: {
		adapter: adapter()
	}
};

export default config;
```

What have I done wrong?

Below is the repository for that project.
https://github.com/tokkitang/tokkitang

Node version is 16, build setting is default.
![image](https://user-images.githubusercontent.com/16988115/227801798-2737fc03-87eb-4d93-a82a-759a60193213.png)

**Comment by teemingc** [comment, marked as answer]
Can you share an example of the error received where ssr isn't working?

**Reply by myyrakle** [reply]
I found the problem and solved it!

The problem was data fetching with the 'ky' function in the 'load' function of .server.ts. Using the fetch object passed as a parameter works fine.

It seems to be limiting outbound inside cloudflare.

**Reply by myyrakle** [reply]
The codepoint where the problem occurred was something like this:
```
export async function load(page: RequestEvent) {
	const accessToken = page.cookies.get('access_token');
	let myUserInfo: User | null = null;
	let isLogin = false;
	let teamList: Team[] = [];
	let error: any = null;

	if (accessToken) {
		try {
			// using ky...
			const userInfoResponse = await getUserInfo(accessToken);
			myUserInfo = { ...userInfoResponse };
			isLogin = true;

			const teamListResponse = await getMyTeamList(accessToken);
			teamList = teamListResponse.list;
		} catch (error) {
			error = error;
			console.error(error);
		}
	}

	return {
		accessToken,
		myUserInfo,
		isLogin,
		error,
		teamList
	};
}
```

**Reply by myyrakle** [reply]
```
Error: The 'credentials' field on 'RequestInitializerDict' is not implemented.
```

This is the error message that occurred when using 'ky'.

Thanks for your interest in my question!
